java/core/src/main/java/co/worklytics/psoxy/gateway/impl/oauth/AccountCredentialsGrantTokenRequestBuilder.java

```diff
@@ -60,23 +61,26 @@ public enum ConfigProperty implements ConfigService.ConfigProperty {
 
     @Override
     public HttpContent buildPayload() {
+
+        //trim to avoid copy-paste errors
+        //q : check for non-printable chars or anything like that
```

Comment:
yeah, I'd include a warning if accountId != trim(accountId)

Reply:
that will do the whitespace before/after, but easy to trim out; my concern is encapsulated stuff, like if they mixed up encodings or something.